test/visreg/tests/modals.ts

```diff
@@ -0,0 +1,25 @@
+import {By, Snappit, WebDriver, Key, WebElement, Browser} from "snappit-visual-regression";
+import {test, TestContext} from "ava";
+import * as util from "../../common/util";
+
+const getModal = async (
+    driver: WebDriver,
+    modalName: string,
+    path: string
+) => {
+    (modalName == "Close Modal") ? await util.$x(driver, util.selectors.openModal, "Open Modal").click() : '';
+    const modal = await util.$x(driver, path);
+    return modal;    
+}
```

Comment:
Why even manage a `"Close Modal"` argument with this ternary logic?

Just drop the whole `getModal` method and directly manage the element in the test body.

Reply:
Thank you Andrew. Removed this logic and made the code simple.